src_assets/common/assets/web/header.html

```diff
@@ -68,6 +68,9 @@
   .nav-link.active {
     font-weight: 500;
   }
+  .form-control::placeholder {
+    opacity: 0.5;
+  }
```

Comment:
This is good enough for now. Eventually I'll clean up this css and separate it from the html code.